### PR TITLE
LPS-151833 Element editor shows the top level JSON only + help text updated

### PR DESCRIPTION
**Pull request by oliv-yu**
https://issues.liferay.com/browse/LPS-151833 - Element editor shows the top level JSON attributes only, help text updated in the sidebar

A bunch of small changes:
- The `elementDefinition` of a default new element will take on the same `elementDefinition` as a custom JSON
- If a description is blank, the placeholder in the editor is just something similar to `{'en_US': ''}`
- Help text was added into the sidebar, regarding Autocompletion of JSON + language keys
- For organizational purposes, the query element that was used as the default for created elements was moved into test files 😺 

Screenshot:
Screen Shot 2022-05-04 at 12 34 13 PM

### CODE REVIEW

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -2232,6 +2232,7 @@ before-the-event-by=before the event by:
 begin-a-workflow=Begin a workflow.
 begin-date=Begin Date
 begin-each-line-with=Begin each line with
+begin-typing-inside-double-quotes-to-see-the-autocompletion-options-or-press-xx=Begin typing inside double quotes to see the autocompletion options, or press xx.
```
Comment (thektan) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:2235]: Currently there's no hotkey to open autocomplete besides typing in the quotes. So I'd leave out the last part, "Begin typing inside double quotes to see the autocompletion options."

```suggestion
begin-typing-inside-double-quotes-to-see-the-autocompletion-options=Begin typing inside double quotes to see the autocompletion options.
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -8217,6 +8218,7 @@ js-line-break=JS Line Break
 js-munge=JS Munge
 js-preserve-all-semicolons=JS Preserve All Semicolons
 js-verbose=JS Verbose
+json-autocompletion=JSON Auto-completion
```
Comment (thektan) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:8221]: No hyphen

```suggestion
json-autocompletion=JSON Autocompletion
```

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -17203,6 +17206,7 @@ use-the-original-author-help=Keep authorship of imported content whenever possib
 use-the-same-assignee-for-all-tasks=Use the same assignee for all tasks.
 use-the-same-look-and-feel-of-the-pages-in-which-this-template-is-used=Use the same look and feel of the pages in which this template is used.
 use-the-same-look-and-feel-of-the-x=Use the same look and feel of the {0}.
+use-the-space-key-to-select-a-menu-item=Use the space key to select a menu item.
```
Comment (thektan) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:17209]:
```suggestion
use-the-enter-key-to-select-a-menu-item=Use the enter key to select a menu item.
```

**Review by thektan — CHANGES_REQUESTED** [review]
Made comments for help text updates to reflect the current behavior.

**Comment by oliv-yu** [comment]
Added those updates to the language keys and rebased! 👍

**Comment by thektan** [comment]
ci:test:sf

**Review by thektan — APPROVED** [review]

**Comment by thektan** [comment]
ci:forward

**Pull request closed without merging**